spec/models/async_delete_mixin_spec.rb

```diff
@@ -125,12 +125,16 @@ def self.should_queue_delete_on_class_with_many_ids
     context "with 3 ems" do
       before do
         @objects, @obj = common_setup(:ems_vmware)
+
+        # Simplify destroying the EMSs so the pause record doesn't have to be
+        # delivered for the common tests to pass
+        @objects.each { |obj| obj.update!(:enabled => false) }
```

Comment:
NOTE: I don't love this, but it allows the common async_delete_mixin specs to run without having to consider the extra `pause!` queue method that destroying an EMS requires.  An alternative could be to deliver every queue message but there is a decent amount of logic in only delivering the related queue items.

Reply:
As this is a spec, we should know exactly what queue items there are right?  That is, you could do `MiqQueue.find_by(stuff).deliver`

Reply:
Well the issue is the tests are shared across a number of different models, where `ExtManagementSystem` is the only one now which emits 2 queue items instead of just the one.